src/core/reference/src/runtime/reference/utils/fft_common.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph/runtime/reference/utils/fft_common.hpp"
+
+#include <algorithm>
+#include <cassert>
+#include <cstddef>
+#include <cstring>
+#include <numeric>
+
+#include "ngraph/check.hpp"
+
+namespace ngraph {
+namespace runtime {
+namespace reference {
+namespace fft_common {
+std::vector<int64_t> reverse_shape(const ngraph::Shape& shape) {
+    std::vector<int64_t> reversed_shape(shape.begin(), shape.end() - 1);
+    std::reverse(reversed_shape.begin(), reversed_shape.end());
+    return reversed_shape;
+}
+
+std::vector<int64_t> compute_strides(const std::vector<int64_t>& v) {
```

Comment:
The same idea for this method. It doesn't look like fft specific method. We can have more common method or local implementation as I proposed for `reverse_shape` function

Reply:
This method is specific for the support of emulation of complex tensors. At this time, OV supports only FFT-like operations as operations with complex tensors, and complex tensors are emulated as real tensors with two-element tuples as elements.